content/proposals.md

```diff
@@ -79,6 +79,19 @@ Please [see
 MSC1779](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/1779-open-governance.md)
 for full details of the project's Guiding Principles.
 
+For the purposes of transparency and to underpin the "greater benefit"
+principle, proposals MUST disclose possible conflicts of interest where
+they might reasonably exist. Authors MAY do this by filling out the
+following checklist and publishing it together with their proposal.
+
+```
+- [ ] I am being paid for working on this proposal by <ORGANIZATION>
+- [ ] I am an employee or member of <ORGANIZATION> which is involved in the Matrix ecosystem
+- [ ] I have another relationship that might appear to be a conflict of interest: <DETAILS>
+- [ ] I have some other type of financial interest in Matrix or this proposal: <DETAILS>
+- [ ] None of the above
```

Comment:
This doesn't include a situation where the employer is being paid by another organisation to develop the proposal

Reply:
I guess for the author, the first checkbox would still be applicable. It's a good question whether companies sponsoring other companies should also be disclosed. This might actually be tricky because those relationships could be subject to contractual privacy agreements.

Reply:
> I guess for the author, the first checkbox would still be applicable

I feel like it would be a good idea to make that explicit

> This might actually be tricky because those relationships could be subject to contractual privacy agreements.

Yeah, however, nothing stops individuals also making these sorts of contractual agreements. And the first line would still apply there i think? So it makes sense for there to be consistency between individuals and employers. I don't know how either would disclose that they have to keep the customer secret though...

Reply:
>> I guess for the author, the first checkbox would still be applicable
>
> I feel like it would be a good idea to make that explicit

How about something like this?

> I am being paid for working on this proposal by \<ORGANIZATION\> (which might itself be funded or sponsored by other organizations)

Reply:
While I appreciate an attempt to identify a conflict of interest that may hide anywhere down the chain of funding, I don't think it's prudent (let alone feasible) to unwrap the chain all the way, particularly in the checklist which is meant to streamline the declaration. The onus is on the author to declare the CoI wherever it sits - no kind of language can cover all specific cases. If the employer is funded by a private equity firm that has vested interest in X (pun intended), declaring that the employer is funded by that specific private equity firm doesn't expose CoI anyway.